README.md

```diff
@@ -1,18 +1,41 @@
 # REDI Sprint 2020 Frontend Development 2, Final Project
 
-Application Name: {app-name}
+Application Name: **Beauty salon website**.
 
-Student Name: {student-name}
+Student Name: **Anna Danielkiewicz**
 
-## {app-name}
+## DESCRIPTION
 
-{short explanation of your application}
+Beauty salon webpage will be a site sreated for a real make-up artist who has FB and a blog but no website. 
+In my opinion it is the project in which I can use the knowledge acquired during the course and also be creative.
 
-## Screenshots
+The **site map** which you see below you can find here: https://octopus.do/k4t06brxa7g
 
-![screen shot](docs/screen_cap.png)
+![screen shot](docs/site_map.png)
 
-## Features:
+## DESIGN
+
+![screen shot](docs/design_home_page_1.png)
```

Comment:
you forgot to include these file in the pull request. Please commit them so they will properly show up in the README.md

Reply:
Yes, should I like commit them in the same branch and everything will be like in this pull request?

Reply:
exactly yes